engine-rest/engine-rest/pom.xml

```diff
@@ -100,6 +100,17 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.sun.xml.bind</groupId>
+      <artifactId>jaxb-impl</artifactId>
+      <scope>runtime</scope>
+    </dependency>
```

Comment:
1: Yes, test scope was indeed sufficient. I also needed to add it in the jaxrs2 module since rest-core wasn't importing it anymore.

2: Good point as well, removed `jaxb-api`, jakarta is imported anyway as you wrote.